src/app/services/org-team.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import { Team } from 'src/app/Interface/TeamInterface'
+import { BackendService } from './backend.service';
+import { map } from 'rxjs/operators'
+import { Router } from '@angular/router';
+import { AngularFirestore, AngularFirestoreDocument } from '@angular/fire/firestore';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class OrgTeamService {
+
+  teamDetails: Team;
+  teamName: string;
+  public teamDocument: AngularFirestoreDocument<Team>
+  public teamDataObservable: Observable<Team>
+
+  constructor(private backendService: BackendService, private router: Router,  public db: AngularFirestore) { }
+
+  getTeamDetails(){
+    var documentPath = 'Organizations/' + this.backendService.getOrganizationDomain() + '/Teams/' + this.teamName;
+    console.log(documentPath);
+    this.teamDocument = this.db.doc<Team>(documentPath);
```

Comment:
Read data on the basis of appkey. Now it works for only one organization, when we have multipe organization then we need to select organization on the bais of appkey